### ISSUE TITLE
Package `file` not installed by RPM

### ISSUE THREAD

**Issue by Yajo** [odoo/odoo#10947]
And that leads to errors like:

```
2016-02-16 11:28:15,470 17 ERROR dbname openerp.addons.document.content_index: Cannot determine mime type.
Traceback (most recent call last):
  File "/opt/odoo/extra-addons/core-addons/document/content_index.py", line 167, in doIndex
    pop = Popen(['file','-b','--mime',fname], shell=False, stdout=PIPE)
  File "/usr/lib64/python2.7/subprocess.py", line 711, in __init__
    errread, errwrite)
  File "/usr/lib64/python2.7/subprocess.py", line 1327, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory
```

Workaround:

```
yum -y install file
```

**Comment by Yenthe666** [comment]
@d-fence this might still be of interest for you.

**Comment by d-fence** [comment]
I think that this one should be closed as it's for 8.0

**Comment by Yenthe666** [comment]
Good catch, I didn't find any version anywhere.  Judging by the commit on OCB, indeed, this is for V8 and should be closed.
Sorry for bothering you 😉
